docker/setup_configuration/data.yaml

```diff
@@ -131,12 +131,6 @@ notifications_abonnementen_config:
       kanalen:
         - naam: autorisaties
 
-sites_config_enable: true
-sites_config:
-  items:
-    - domain: example.com
-      name: Open Notificaties
```

Comment:
could you undo this change? Since we're not removing sites from this component yet